core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStore.java

```diff
@@ -69,12 +69,14 @@ public void put(PageId pageId, byte[] page) throws IOException {
   }
 
   @Override
-  public ReadableByteChannel get(PageId pageId) throws IOException, PageNotFoundException {
+  public ReadableByteChannel get(PageId pageId, int pageOffset)
+      throws IOException, PageNotFoundException {
     Path p = getFilePath(pageId);
     if (!Files.exists(p)) {
       throw new PageNotFoundException(p.toString());
     }
     FileInputStream fis = new FileInputStream(p.toFile());
+    fis.skip(pageOffset);
```

Comment:
[ERROR] alluxio.client.file.cache.store.LocalPageStore.get(PageId, int) may fail to clean up java.io.InputStream on checked exception [alluxio.client.file.cache.store.LocalPageStore, alluxio.client.file.cache.store.LocalPageStore] Obligation to clean up resource created at LocalPageStore.java:[line 78] is not dischargedPath continues at LocalPageStore.java:[line 79] OBL_UNSATISFIED_OBLIGATION_EXCEPTION_EDGE

Reply:
Fixed.